### DISCUSSION TITLE
Support storing metadata to database for remote videos

### DISCUSSION THREAD

**Discussion by aaronlin0122** [muxinc/next-video#143] [Ideas]
Currently, using remote videos will download the metadata and save to `/videos`, it will be good if it can save the metadata to database by using Prisma or something.

**Comment by heff** [comment]
Thanks for the suggestion! CMSs and alternative databases are on our radar as a next step for the project. We'll wrap in remote videos and consider Prisma as part of that.

**Comment by luwes** [comment]
I did some research on this a while back and after discussions internally a good first step would be:

- [ ] Integrate Drizzle ORM w/ 1 DB type (e.g. [SQLite](https://www.sqlite.org/))

related links:
https://github.com/muxinc/next-video/discussions/199#discussioncomment-8711848
https://authjs.dev/getting-started/adapters

**Reply by lpbonomi** [reply]
Hi @luwes , I've created a draft PR #204 to see if this was more or less what you expected for a first PR for this issue. Please keep in mind this is my first time using Drizzle, I'm more familiarized with Prisma
